js/src/toast.js

```diff
@@ -157,6 +159,10 @@ class Toast extends BaseComponent {
     }
 
     EventHandler.off(this._element, EVENT_CLICK_DISMISS)
+    EventHandler.off(this._element, EVENT_MOUSEOVER)
```

Comment:
You can replace all of them, using just `EventHandler.off(this._element, EVENT_KEY)` ;)